cfme/middleware/provider.py

```diff
@@ -26,7 +25,7 @@
             }],
         'middleware_provider_detail':
         [
-            lambda ctx: list_tbl.click_cells({'name': ctx['provider'].name}),
+            lambda ctx: sel.click(Quadicon(ctx['provider'].name, None)),
```

Comment:
qtype is optional (https://github.com/RedHatQE/cfme_tests/blob/master/cfme/web_ui/__init__.py#L2227), but if you want to specify it properly, it would be "middleware" as you can see couple of lines up. However, middleware quads have no fields so just not specifying the qtype is fine.